scripts/set-backend-env-vars.sh

```diff
@@ -0,0 +1,34 @@
+
+set -o allexport
+[[ -f ./tdrs-backend/.env.production ]] && source ./tdrs-backend/.env.production
```

Comment:
Does this file get created somewhere else?

Reply:
You are expected to provide it

Reply:
Gotcha, I see that now in the testing section. We'll want to work out an automated way of pulling this from the CircleCI environment variables but that can be done with the CI/CD PR.

Reply:
Created this as a stub to track the CircleCI changes we'll need to make with a later PR: https://github.com/raft-tech/tanf-app/issues/844